chord_metadata_service/mcode/api_views.py

```diff
@@ -0,0 +1,57 @@
+from rest_framework import viewsets
+from rest_framework.settings import api_settings
+from .serializers import *
+from chord_metadata_service.restapi.api_renderers import (
+    PhenopacketsRenderer
+)
+from chord_metadata_service.restapi.pagination import LargeResultsSetPagination
+
+
+class McodeModelViewSet(viewsets.ModelViewSet):
+    pagination_class = LargeResultsSetPagination
+    renderer_classes = (*api_settings.DEFAULT_RENDERER_CLASSES, PhenopacketsRenderer)
+
+
+class GeneticVariantTestedViewSet(McodeModelViewSet):
+    queryset = GeneticVariantTested.objects.all().order_by("id")
```

Comment:
Bit of a nitpick but I'd say the default ordering would be better located in the model itself (https://docs.djangoproject.com/en/3.0/ref/models/options/#ordering), possibly in a mixin if we keep using the same field to order by.